features/offer_conditions.feature

```diff
@@ -0,0 +1,64 @@
+@provider
+Feature: Managing conditions on offers
+  Teacher training offers often include conditions that the applicant must fulfil
+  before they can take their place on the course.
+
+  Conditions can be both academic and non-academic. For example, if a candidate
+  still needs to sit exams in order to obtain necessary qualifications or
+  undergo the required criminal record checks to provide evidence of suitability
+  to teach. The provider isn't allowed to make a successful interview a condition
+  (i.e. an interview should happen before and offer is made, not after).
+
+  Background:
+    Given the following providers:
+      | provider code | is an accredited body? |
```

Comment:
The "accredited body" concept is new here — should we add a note about what accredited bodies are in the feature description?

Reply:
Thanks - fixed on #186